src/adapters/bidderFactory.js

```diff
@@ -249,7 +250,7 @@ export function newBidder(spec) {
               typeof request.data === 'string' ? request.data : JSON.stringify(request.data),
               {
                 method: 'POST',
-                contentType: 'text/plain',
+                contentType: request.contentType || 'text/plain',
                 withCredentials: true
               }
```

Comment:
Ony minor thing... how do you feel about `request.options.contentType`, where `options` is an object?

Reason being: this could become `Object.assign({}, theseDefaultPostOptions, requests.options)`, and then bidders will be able to edit any of the `ajax` options that they wanted.

Another benefit is that the JSDoc types would be reusable. The fourth argument to `ajax` would have the same `typedef as the `ServerRequest.options` object.

Reply:
Makes sense to me. Should the `Object.assign()` route only be implemented/tested for the POST ajax call? Or would there be a reason to pass options on the GET too?

Reply:
hmm... I guess I'd have a small preference towards doing it for both, just because parallelism makes the behavior easier for people to remember.

It's a pretty weak preference, though. At worst, the first adapter which passes options to GET will find out that it doesn't work, and then they'll just have to add it later.